sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2151,6 +2158,9 @@ class SQLConf extends Serializable with Logging {
   def setCommandRejectsSparkCoreConfs: Boolean =
     getConf(SQLConf.SET_COMMAND_REJECTS_SPARK_CORE_CONFS)
 
+  def maxSqlForEventLength: Int =
```

Comment:
In general, function name should be matched to configuration name. Please update the variable name and function name together consistently.

Reply:
i have removed it as we can use StaticSQLConf reference directly